Makefile

```diff
@@ -15,27 +15,20 @@ e2e_build_images: e2e_clean_slate
 	@docker build -t ethereum:prebuilt -f integration_tests/ethereum/Dockerfile integration_tests/ethereum/
 
 e2e_clean_slate:
-	@docker rm --force \
-		$(shell docker ps -qa --filter="name=ethereum") \
-		$(shell docker ps -qa --filter="name=sommelier") \
-		$(shell docker ps -qa --filter="name=orchestrator") \
-		$(shell docker ps -qa --filter="name=steward") \
-		1>/dev/null \
-		2>/dev/null \
-		|| true
-	@docker wait \
-		$(shell docker ps -qa --filter="name=ethereum") \
-		$(shell docker ps -qa --filter="name=sommelier") \
-		$(shell docker ps -qa --filter="name=orchestrator") \
-		$(shell docker ps -qa --filter="name=steward") \
-		1>/dev/null \
-		2>/dev/null \
-		|| true
-	@docker network prune --force 1>/dev/null 2>/dev/null || true
-	@cd integration_tests && go test -c
-
-e2e_cork_test: e2e_clean_slate
-	@E2E_SKIP_CLEANUP=true integration_tests/integration_tests.test -test.failfast -test.v -test.run IntegrationTestSuite -testify.m TestCork || make -s fail
+	@./clean_slate.sh
+
+e2e_cork_test: e2e_aave_v2_stablecoin_test e2e_vault_cellar_test
```

Comment:
If we're going to do this, we should update the integration test action to call the two different integration tests separately so they could be parallelized.